src/main/kotlin/coverosR3z/authentication/AuthenticationUtilities.kt

```diff
@@ -16,37 +16,26 @@ class AuthenticationUtilities(val ap : IAuthPersistence, private val cua : ICurr
     /**
      * Register a user through auth persistent, providing a username, password, and employeeId
```

Comment:
should probably note that employeeId does not in fact need to be provided when calling this method